1443/en.md

```diff
@@ -0,0 +1,121 @@
+# LOJ 1443 - Printing Divisors
+
+## Summary
+Print the divisors of a given number N in an order such that when two consecutive divisors are divided, the result is a prime number.
+
+## Prerequisite
+Prime factorization: https://cp-algorithms.com/algebra/factorization.html
+(For solving this problem the basic O( $\sqrt{N}$ ) algorithm will suffice)
+
+## Solution
+Say, N = $2^4$ * $3^2$ * $5^2$
+
+M = $2^4$ * $3^0$ * $5^0$
+
+Clearly M is a divisor of N and writing all the divisors of M is pretty easy:-
+
+
+1, $2^1$, $2^2$, $2^3$, $2^4$ = M. </span>
+
+Now, let's raise the power of 3 by 1. M = $2^4$ * $3^1$ * $5^0$
+
+<span style="color:aqua"> 1, $2^1$, $2^2$, $2^3$, $2^4$</span>, 
+
+<span style="color:lime">$2^4$ * $3^1$ = M, $2^3$ * $3^1$, $2^2$ * $3^1$, $2^1$ * $3^1$, 1 * 3.</span>
+
+Raising the power of 3 again by 1. M = $2^4$ * $3^2$ * $5^0$
+
+<span style="color:aqua"> 1, $2^1$, $2^2$, $2^3$, $2^4$</span>, 
+
+<span style="color:lime">$2^4$ * $3^1$, $2^3$ * $3^1$, $2^2$ * $3^1$, $2^1$ * $3^1$, 1 * 3</span>,
+
+<span style="color:violet">1 * $3^2$, $2^1$ * $3^2$, $2^2$ * $3^2$, $2^3$ * $3^2$, $2^4$ * $3^2$ = M.</span>
+
+Raising the power of 5 by 1. M = $2^4$ * $3^2$ * $5^1$
+
+<span style="color:aqua"> 1, $2^1$, $2^2$, $2^3$, $2^4$, $2^4$ * $3^1$, $2^3$ * $3^1$, $2^2$ * $3^1$, $2^1$ * $3^1$, 1 * 3, 1 * $3^2$, $2^1$ * $3^2$, $2^2$ * $3^2$, $2^3$ * $3^2$, $2^4$ * $3^2$</span>, 
+
+<span style="color:lime"> $2^4$ * $3^2$ * $5^1$ = M, $2^3$ * $3^2$ * $5^1$, $2^2$ * $3^2$ * $5^1$, $2^1$ * $3^2$ * $5^1$, $3^2$ * $5^1$, $3^1$ * $5^1$, 1 * $5^1$.</span>
+
+Now, M = N = $2^4$ * $3^2$ * $5^2$
+
+<span style="color:aqua"> 1, $2^1$, $2^2$, $2^3$, $2^4$, $2^4$ * $3^1$, $2^3$ * $3^1$, $2^2$ * $3^1$, $2^1$ * $3^1$, 1 * 3, 1 * $3^2$, $2^1$ * $3^2$, $2^2$ * $3^2$, $2^3$ * $3^2$, $2^4$ * $3^2$</span>, 
+
+<span style="color:lime"> $2^4$ * $3^2$ * $5^1$, $2^3$ * $3^2$ * $5^1$, $2^2$ * $3^2$ * $5^1$, $2^1$ * $3^2$ * $5^1$, $3^2$ * $5^1$, $3^1$ * $5^1$, 1 * $5^1$ </span> , 
+
+<span style="color:violet"> 1 * $5^2$, $3^1$ * $5^2$, $3^2$ * $5^2$, $2^1$ * $3^2$ * $5^2$, $2^2$ * $3^2$ * $5^2$, $2^3$ * $3^2$ * $5^2$, $2^4$ * $3^2$ * $5^2$ = M.</span>
```

Comment:
Instead of having `= M` at the very end of the divisors' list (sometimes in the middle as well, mistakenly), I would instead like there to be some kind of prompt or a dialogue saying something like "The divisors of M now are:".

Reply:
fixed!